File: react/src/api/models/acroStack/services/dtos/backgroundJobs/BackgroundJobDto.ts
/* oxlint-disable */

/**
 * @type object
 */
export type AcroStackServicesDtosBackgroundJobsBackgroundJobDto = {
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  id?: string;
  /**
   * @type string | undefined
   */
  jobName?: string | null;
  /**
   * @type string | undefined
   */
  jobArgs?: string | null;
  /**
   * @description
   * Format: `int32`
   * @type integer | undefined
   */
  tryCount?: number;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  creationTime?: string;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  nextTryTime?: string;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  lastTryTime?: string | null;
  /**
   * @type boolean | undefined
   */
  isAbandoned?: boolean;
};

File: src/AcroStack/Services/Dtos/BackgroundJobs/BackgroundJobDto.cs
using System;

namespace AcroStack.Services.Dtos.BackgroundJobs;

public class BackgroundJobDto
{
    public Guid Id { get; set; }
    public string? JobName { get; set; }
    public string? JobArgs { get; set; }
    public short TryCount { get; set; }
    public DateTime CreationTime { get; set; }
    public DateTime NextTryTime { get; set; }
    public DateTime? LastTryTime { get; set; }
    public bool IsAbandoned { get; set; }
}
